src/test/resources/features_unitary/functionality/example_unitary.feature

```diff
@@ -0,0 +1,34 @@
+Feature: sample karate test script
+  for help, see: https://github.com/intuit/karate/wiki/IDE-Support
+
+  Background:
+    * url 'https://jsonplaceholder.typicode.com'
```

Comment:
maybe change this url to a variable here so the example is also using that for api urls as we should be using on every feature then

Reply:
corrected